rust/private/utils.bzl

```diff
@@ -179,6 +179,23 @@ def expand_locations(ctx, env, data):
     """
     return dict([(k, _expand_location(ctx, v, data)) for (k, v) in env.items()])
 
+def expand_arg_locations(ctx, args, data):
+    """Performs location-macro expansion on a list of string values.
+
+    See `expand_locations` for detailed documentation.
```

Comment:
Please mention `$(execroot ...) and $(location ...) are prefixed with ${pwd},` there (to keep symmetry with `expand_env_locations`.